### DISCUSSION TITLE
Feature Request: Some indication when a car is 3 seconds or less behind another car

### DISCUSSION THREAD

**Discussion by jtokash** [f1multiviewer/issue-tracker#306] [Feature requests]
The new dark green background indicating that they are in DRS range is great, but the green outline meant to highlight that they are chasing isn't great.  In my experience watching the Brazil race, it seems to highlight any driver that has a lap time lower than the car in front of it - even if the gap interval is 16.745.  Therefore it doesn't highlight drivers that are 2 seconds behind if their previous lap time was not faster than the car ahead.

I tried to remove spoilers from this screenshot, but view with caution if you haven't watched the race.

<img width="1835" alt="spoilersremoved" src="https://user-images.githubusercontent.com/614952/201559888-29833753-01db-4e32-8c08-c906a903431b.png">

**Comment by f1multiviewer** [comment]
It is supposed to show if the driver is "catching" the driver ahead, so if the gap has, on average, decreased. It's something F1 calculate so it's not something I can comment on in detail, but I'd assume they have a rudimentary algorithm to decide this.

**Reply by nicoabie** [reply]
They are using AWS probably and we know how that went. They had to remove the overtaking difficulty haha

**Comment by jtokash** [comment]
That makes sense, but I would still love some indication that the gap is 3 seconds or less.  Arbitrary? Maybe!
